### ISSUE TITLE
A11y_SQLdatabase projects extension_Create New Database Project_Type - Azure SQL database_Voiceover: Voiceover is not announcing the 'Type' group as required.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#22127]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #WCAG1.3.1; #MAC; #AINotInScope; #Info and Relationships; #Voiceover; #Element:button;

### Actual Result:​
Voiceover is not announcing the 'Type' group as required.
Type group is marked as required but voiceover is not reading as type group required.

**Note issue:**

1. Open existing Project - Location : Voiceover is not announcing as required for location radio group
2. Create New Database Project : Voiceover is not announcing as required for server and location controls.
3. Publish target - New SQL server : Voiceover is not announcing as required for below controls

- Azure SQL server port number
- Azure SQL server admin password
- Confirm Azure SQL server admin password

### Expected Result:​
Voiceover should be announcing the 'Type' group as required.
Type group is marked as required so voiceover should read as type group required.

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1. Start Voiceover with using shortcut key Command+F5.
2. Launch Azure Data Studio
5. Tab till extension and search for “SQL Database Projects” and install it.
6. Tab till database project and press enter.
7. “Database Project” panel will open.
8. Tab till “Create New” button and press enter.
9. “Create New Database Project” window will open(default type will set to Azure SQL database).
10. verify whether voiceover is announcing the 'Type' group as required or not.

### User Impact:
Voiceover user will miss required control information and face difficulties while creating project, if voiceover does not announce as required for required control.

### Attachment:
![Type group2](https://user-images.githubusercontent.com/86241494/222658858-0ec7efe7-77d2-434c-85b4-6c2a7629aa9b.gif)

**Note issue:**
![Location required](https://user-images.githubusercontent.com/86241494/223633552-1761bd9f-de5e-4a5e-be35-9cf9e12360bf.gif)
![Server and location not announcing as required](https://user-images.githubusercontent.com/86241494/223633559-e1b0983f-7ebf-48fe-b21e-14ae3d328c88.gif)
![publish to new sql - not announcing required](https://user-images.githubusercontent.com/86241494/223635948-acfabf88-8a53-4b98-927e-2eb59e72b812.gif)

**Comment by kisantia** [comment]
@SachinChavhan317 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by kisantia** [comment]
@SachinChavhan317 Please verify this issue is fixed.

**Comment by SachinChavhan317** [comment]
Verified the bug on Version: 1.44.0-insider (Universal)
Issue is fixed for three controls. 
For location control issue is still reproducing.

Please refer below attachment for reference.

Repro attachment:
![Location still not announcing as fixed](https://user-images.githubusercontent.com/86241494/233352427-f0a55bf3-c46d-4c26-abe6-29a1abab626a.gif)

Fixed attachments
![22127_Fixed](https://user-images.githubusercontent.com/86241494/233352375-db2e98e8-a79a-4f0e-83a0-2aeacd26f6bd.gif)
![22127_Fixed2](https://user-images.githubusercontent.com/86241494/233352575-470c982e-996f-4b2b-9d33-8d19a335a907.gif)
![22253_Type](https://user-images.githubusercontent.com/86241494/233352818-a0b0bc01-1c40-424e-9ca8-f43f4f0beeb2.gif)

**Comment by kisantia** [comment]
@SachinChavhan317 please verify this is fixed and close the issue. The "Open Existing Project" dialog no longer has the required indicator for the Location radio buttons

**Comment by SachinChavhan317** [comment]
Verified the bug on below version,

Version: 1.47.0-insider (Universal)
Commit: a63463e35bb21bb89250be09d9639ff440c83b39
Date: 2023-09-21T05:42:38.696Z
VS Code: 1.79.2
Electron: 22.3.14
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.6.0


The "Open Existing Project" dialog no longer has the required indicator for the Location radio buttons. Hence closing the bug. #Closed;
<img width="1440" alt="Screenshot 2023-09-25 at 5 27 38 PM" src="https://github.com/microsoft/azuredatastudio/assets/86241494/8c52387a-5dce-4931-9e2a-08977e7fc794">
